tests/runtests.py

```diff
@@ -391,8 +408,8 @@ def paired_tests(paired_test, options, test_labels, parallel):
              'LiveServerTestCase) is expected to run from. The default value '
              'is localhost:8081-8179.')
     parser.add_argument(
-        '--selenium', action='store_true', dest='selenium', default=False,
-        help='Run the Selenium tests as well (if Selenium is installed).')
+        '--selenium', dest='selenium', default=None,
```

Comment:
I feel `selenium=firefox,chrome` would be more convenient than `--selenium=firefox --selenium=chrome`.